arbnode/resource_management.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2023, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package arbnode
```

Comment:
I think this should be a separate package, arbnode package is already blown up too much with things that should be in separate packages.

Reply:
Good idea, I moved it to `arbnode/resourcemanager`